kamon-logback-stackdriver/src/test/scala/kamon/stackdriver/StackdriverEncoderTest.scala

```diff
@@ -1,33 +1,31 @@
 package kamon.stackdriver
 
-import ch.qos.logback.classic.Level
-import ch.qos.logback.classic.spi.{ILoggingEvent, LoggingEvent}
 import kamon.Kamon
 import kamon.context.Context
-import kamon.tag.TagSet
+import kamon.stackdriver.StackdriverMarker.LogValue.{NestedValue, StringValue}
 import kamon.trace.Trace.SamplingDecision
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
-import org.slf4j.{Logger, LoggerFactory}
+import org.slf4j.{Logger, LoggerFactory, Marker}
 import spray.json.DefaultJsonProtocol._
 import spray.json._
 
 class StackdriverEncoderTest extends AnyFlatSpec with Matchers {
-
   Kamon.init()
 
   val logger: Logger = LoggerFactory.getLogger(classOf[StackdriverEncoderTest])
 
   it should "format correctly log as json" in {
 
-    val contextEntry = Context.key[String]("context_entry", "default_context_entry_value")
+    val marker: Marker = BasicStackdriverMarker("audit", NestedValue(Map("foo" -> StringValue("bar"))))
```

Comment:
probably it will be worth to show and test merging of tokens

Reply:
what do you mean by merging tokens? The `BasicStackdriverMarker` has it's own test suite